### PR TITLE
feat: delete https certificate file

### LINKED ISSUE #135: Delete HTTPS certificate file
**What would you like to be added**:

delete https certificate files

**Why is this needed**:

for safety reasons

### PR DESCRIPTION
**Pull request by OrangeBao**
#### What type of PR is this?

feat
#### What does this PR do?

delete https certificate
#### Which issue(s) does this PR fix?

Fixes #135 
#### Screenshots (if applicable):

```release-note

```

#### Additional Comments:

```release-note
delete https certificate file

```

### CODE REVIEW

File: cmd/clusterlink/floater/certificate/file.key [hunk 1]
```diff
@@ -1,27 +1 @@
...
-4ocmiQRmlDYnuvkKWWcsEJ4cWXig3KChuUX/QHhGzmbRls//vyiGc65trngrny4Z
-4bD7EN+FhfIa9ecPXqeVupZ4voN7wr73DF3wExKuZfixYjYp/hXsMoOkHtZ+Tjph
-Q58ZfGHuLqSZMTTCBnikoQ8CgYAbT6lCsZ7invx6p1ABncFOA+bINjgn1AStsr6R
-LrdIUgsVCZt99+NlCqU9FoGVGpdyzZPRt9e4kqUd21J2Jubb/SS5+8TeZ6N704cG
-dmOsdWGLPzO6FnAIJVo+iLeMffRxQZCjyY/TSx8VlWuZcZrmd7tbSvCc1iJFB8R0
-vnqpBQKBgQDiZY+lSVdEVsdIygsi7AeReSQnI66ZElO64QrE1s116zDmIvvDJwpT
-+8WFNvrt33PLrpbDgkPu6cenWGysl/munvfwwmULUZ+hddbxv2TUX7WvDnWMnfw8
-NuW0aG4YFjkyzGwKsWkMkKYZ2l9Ls4SYcnxsc+EfbhiQj5GrlPJgPQ==
------END RSA PRIVATE KEY-----
+<<Please generate the certificate file yourself>>
```
Comment (duanmengkk) [inline cmd/clusterlink/floater/certificate/file.key]: add a new line

Comment (OrangeBao) [reply cmd/clusterlink/floater/certificate/file.key]: ok. I get it.

**Comment by wuyingjun-lucky** [comment]
/hold
@OrangeBao you might rebase  the commits

**Comment by OrangeBao** [comment]
The service `floater` is only for testing purposes. Using secret to mount certificates increases complexity.
So I replaced https with http.

**Pull request merged**